### PR TITLE
Add Japanese translation of release note 1.5.0

### PR DESCRIPTION
**Pull request by shino**
And remove an item which was closed without merge, both from en/ja.

English lines will be removed after +1'ed.

### CODE REVIEW

File: RELEASE-NOTES.ja.md [hunk 1]
```diff
@@ -1,3 +1,142 @@
...
+* `ERL_MAX_PORTS` を Riak のデフォルトに合わせ 64000 へ変更 [riak_cs/#636](https://github.com/basho/riak_cs/pull/636)
+* Allow Riak CS admin resources to be used with OpenStack API [riak_cs/#666](https://github.com/basho/riak_cs/pull/666)
+* Riak CS 管理リソースを OpenStack API でも利用可能にする修正 [riak_cs/#666](https://github.com/basho/riak_cs/pull/666)
+* Fix path substitution code to fix Solaris source builds [riak_cs/#733](https://github.com/basho/riak_cs/pull/733)
+* Solaris でのソースビルドのバグ修正のため、パス代入コードの変更 [riak_cs/#733](https://github.com/basho/riak_cs/pull/733)
+* `sanity_check(true,false)` logs invalid error on `riakc_pb_socket` error [riak_cs/#683](https://github.com/basho/riak_cs/pull/683)
+* `riakc_pb_socket` エラー時の `sanity_check(true,false)` バグを修正 [riak_cs/#683](https://github.com/basho/riak_cs/pull/683)
+* Riak-CS-GC timestamp for scheduler is in the year 0043, not 2013. [riak_cs/#713](https://github.com/basho/riak_cs/pull/713) fixed by [riak_cs/#676](https://github.com/basho/riak_cs/pull/676)
+* Riak-CS-GC のスケジューラタイムスタンプが 2013 ではなく 0043 になるバグを修正. [riak_cs/#713](https://github.com/basho/riak_cs/pull/713) fixed by [riak_cs/#676](https://github.com/basho/riak_cs/pull/676)
```
Comment (kuenishi) [inline RELEASE-NOTES.ja.md]: one dot here `.`

File: RELEASE-NOTES.ja.md [hunk 2]
```diff
@@ -1,3 +1,142 @@
...
+* Excessive calls to OTP code_server process #669 fixed by [riak_cs/#675](https://github.com/basho/riak_cs/pull/675)
+* OTP code_server プロセスへの過大な呼び出しを修正 [riak_cs/#675](https://github.com/basho/riak_cs/pull/675)
```
Comment (kuenishi) [inline RELEASE-NOTES.ja.md]: I prefer `過剰` or `過度`. Whichever. 
"code_server プロセスの過度の呼び出し" : 「へ」 is strange to me.

File: RELEASE-NOTES.ja.md [hunk 3]
```diff
@@ -1,3 +1,142 @@
...
+* Return HTTP 400 if content-md5 does not match [riak_cs/#596](https://github.com/basho/riak_cs/pull/596)
+* content-md5 が一致しない場合に HTTP 400 を返すよう修正 [riak_cs/#596](https://github.com/basho/riak_cs/pull/596)
+* `/riak-cs/stats` and `admin_auth_enabled=false` don't work together correctly. [riak_cs/#719](https://github.com/basho/riak_cs/pull/719)
+* `/riak-cs/stats` が `admin_auth_enabled=false` の時に動作しなバグを修正. [riak_cs/#719](https://github.com/basho/riak_cs/pull/719)
```
Comment (kuenishi) [inline RELEASE-NOTES.ja.md]: しなバグ

File: RELEASE-NOTES.ja.md [hunk 4]
```diff
@@ -1,3 +1,142 @@
...
+* Storage calculation doesn't handle tombstones, nor handle undefined manifest.props [riak_cs/#849](https://github.com/basho/riak_cs/pull/849)
+* ストレージ計算で tombstone および undefined の manifest.props を処理できないバグを修正 [riak_cs/#849](https://github.com/basho/riak_cs/pull/849)
+* MP initiated objects remains after delete/create buckets #475 fixed by [riak_cs/#857](https://github.com/basho/riak_cs/pull/857) and [stanchion/#78](https://github.com/basho/stanchion/pull/78)
+* 開始されたマルチパートオブジェクトが、バケットの削除、作成後にも残るバグを修正 [riak_cs/#857](https://github.com/basho/riak_cs/pull/857) and [stanchion/#78](https://github.com/basho/stanchion/pull/78)
```
Comment (kuenishi) [inline RELEASE-NOTES.ja.md]: 「未完了のマルチパート…」 is natural to me

File: RELEASE-NOTES.ja.md [hunk 5]
```diff
@@ -1,3 +1,142 @@
...
+* 匿名アクセス時の不要なログを削除 [riak_cs/#876](https://github.com/basho/riak_cs/issues/876)
+* Fix inconsistent ETag on objects uploaded by multipart [riak_cs/#855](https://github.com/basho/riak_cs/issues/855)
+* マルチパートでアップロードされたオブジェクトの ETag 不正を修正 [riak_cs/#855](https://github.com/basho/riak_cs/issues/855)
+* Fix policy version validation in PUT Bucket Policy [riak_cs/#911](https://github.com/basho/riak_cs/issues/911)
+* PUT Bucket Policy のポリシーバージョン確認の不具合を修正[riak_cs/#911](https://github.com/basho/riak_cs/issues/911)
+* Fix return code of several commands, to return 0 for success [riak_cs/#908](https://github.com/basho/riak_cs/issues/908)
+* コマンド成功時に終了コード 0 を返すよう修正 [riak_cs/#908](https://github.com/basho/riak_cs/issues/908)
+
+
+## Notes on Upgrading
```
Comment (kuenishi) [inline RELEASE-NOTES.ja.md]: Won't this be translated?

Comment (shino) [inline RELEASE-NOTES.ja.md]: Shame on me :disappointed: Most important parts. I will.

**Comment by shino** [comment]
Addressed comments and translated parts I forgot... :sweat_smile: 
Please review m(＿ ＿)m

**Comment by shino** [comment]
@borshop merge

**Pull request merged**